AndelaOSP/art-dashboard
# Fix: fix pagination on users, assets and asset types pages

## What does this PR do?
Fixes pagination on users, assets and asset types pages
## Description of Task to be completed?
- Add `page_size` query parameters to the urls in the actions.
- Add `page_size` query parameters to respective test files
## How should this be manually tested?
- Click the Assets, Users or Asset Types menu links
- On the displayed table, click on all pages on the pagination.
- Observe that data from the backend is displayed on all pages
## What are the relevant pivotal tracker stories?
[159132708](https://www.pivotaltracker.com/story/show/159132708)
## Any background context you want to add?
N/A
## Important notes
N/A
## Packages installed
N/A
## Deployment note
N/A
## Related PRs branch | PR (branch_name) | (pr_link)
N/A

## Todos
## Screenshots
<img width="1409" alt="screen shot 2018-07-18 at 12 48 05 pm" src="https://user-images.githubusercontent.com/26274319/42879537-ed41edd6-8a88-11e8-9b63-b53652975df4.png">
<img width="1367" alt="screen shot 2018-07-18 at 12 48 22 pm" src="https://user-images.githubusercontent.com/26274319/42879538-ed69394a-8a88-11e8-90fd-65c4c1a89d96.png">
<img width="1355" alt="screen shot 2018-07-18 at 12 48 35 pm" src="https://user-images.githubusercontent.com/26274319/42879539-eda3e892-8a88-11e8-9f9a-b405857dab16.png">
